src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -85,6 +87,15 @@ public async Task<IEnumerable<IAccount>> GetAccountsAsync()
         {
             RequestContext requestContext = CreateRequestContext(Guid.NewGuid());
             IEnumerable<IAccount> accounts = Enumerable.Empty<IAccount>();
+
+#if ANDROID
+            if (AppConfig.IsBrokerEnabled)
+            {
+                var broker = ServiceBundle.PlatformProxy.CreateBroker(null);
```

Comment:
>null [](start = 70, length = 4)

it's okay to have a null broker here?

Reply:
it is not a null broker, the activity that is passed in is null which is fine for non interactive broker operations

Reply:
I think the iOS broker doesn't support this. You should add an extra condition to the `if` on line 92: 

`if AppConfig.BrokerEnabled && PlatformProxy.BrokerSupportsSilentCall `

`GetAccounts` does not change for iOS broker.

Reply:
ah, i for got about this parameter. yes it will be cleaner